ncm-pam/src/main/pan/components/pam/config.pan

```diff
@@ -114,7 +114,7 @@ function pam_add_listfile_acl = {
     ret = pam_add(service, pamtype, control, "listfile", opts);
     # Now, grab the entry that was just put at the end of the list and
     # add in the ACL information.
-    ret[service][pamtype][length(ret[service][pamtype])-1][sense] = dict("filename", filename, "items", items);
+    ret[service][pamtype][length(ret[service][pamtype]) - 1][sense] = dict("filename", filename, "items", items);
```

Comment:
didn;t we add support for negative indices in pan a while back?

Reply:
So we did!